crates/bevy_winit/src/winit_windows.rs

```diff
@@ -167,7 +167,7 @@ impl WinitWindows {
         );
         adapters.insert(entity, adapter);
         handlers.insert(entity, handler);
-        winit_window.set_visible(true);
+        winit_window.set_visible(window.visible);
```

Comment:
I think you should use `with_visible` (https://docs.rs/winit/latest/winit/window/struct.WindowBuilder.html#method.with_visible) on the WindowBuilder at some point before line 146 (`let winit_window = winit_window_builder.build(event_loop).unwrap();`) ? This would probably solve the white window thing (which I think is not solved yet?)

Reply:
I did it that way because I didn't want to change the existing behaviour. It seems to have been done this way for the accesskit PR so I pinged the author to know why they did that.

Also, I tried using with_visible and it didn't fix the white flash, I do agree that it seems more like the intended api for this.